### DISCUSSION TITLE
Not working in ventura using SIP enabled

### DISCUSSION THREAD

**Discussion by StebanPi** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#300] [General]
I'm using ventura, when i enabled the sip, i had already used the notarized, it gives me the error StatusBarApp Is damaged, bit when i start with the sip dissbled ir doesn't give me any error.
Please help!!
![16865303657416312263082053137386](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/92879371/2064ae23-1417-479c-860a-42987d959c9a)

**Comment by chris1111** [comment]
[Read the rules](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/discussions/19) before posting a discussion
